Plugins/Flow.Launcher.Plugin.BrowserBookmark/Flow.Launcher.Plugin.BrowserBookmark.csproj

```diff
@@ -54,22 +55,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="Languages\*.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
```

Comment:
why are we removing this, does bookmark plugin not have any language files?

Reply:
Because xaml is included by default for wpf program

Reply:
cool thanks for letting me know. How come we dont do this for other projects?

Reply:
Because these two are error when building🤣

Reply:
Some of the plugin doesn't have a setting page, so doesn't have to enable wpf

Reply:
I think a couple more plugins do have settings and use wpf, could you make the change so it's consistent while you are making the others please

Reply:
hold on, are you sure it's included by default?
![image](https://user-images.githubusercontent.com/26427004/108828239-b7371600-761a-11eb-94c8-76441f0f4656.png)

Reply:
sure